### PR TITLE
add random forest assignment

### PR DESCRIPTION
**Pull request by LongfeiH**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] [local build the book](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [x] review the artifacts from local build
- [x] [local build the slides](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [x] Unit test if appliable

# Checklist:

- [x] My code follows the [style guidelines](https://github.com/open-academy/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have applied tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: open-machine-learning-jupyter-book/assignments/ml-advanced/ensemble-learning/random-forests-intro-and-regression.ipynb [hunk 1]
```diff
@@ -0,0 +1,1325 @@
+{
```
Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-advanced/ensemble-learning/random-forests-intro-and-regression.ipynb:1]: <pre class="ql-syntax" spellcheck="false">%matplotlib inline
</pre><p> </p><p>put this line to the first</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-advanced/ensemble-learning/random-forests-intro-and-regression.ipynb:1]: <p>remove line 13 rmse</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-advanced/ensemble-learning/random-forests-intro-and-regression.ipynb:1]: <p>The rubric table is not rendering. </p><p> </p><p>Seems that this Rubric section can be removed safely. It doesn't serve us.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (LongfeiH) [reply open-machine-learning-jupyter-book/assignments/ml-advanced/ensemble-learning/random-forests-intro-and-regression.ipynb:1]: done

Comment (LongfeiH) [reply open-machine-learning-jupyter-book/assignments/ml-advanced/ensemble-learning/random-forests-intro-and-regression.ipynb:1]: done

Comment (LongfeiH) [reply open-machine-learning-jupyter-book/assignments/ml-advanced/ensemble-learning/random-forests-intro-and-regression.ipynb:1]: II have removed it

**Review by reveurmichael — APPROVED** [review]

**Pull request merged**